template.rb

```diff
@@ -61,6 +61,10 @@ def apply_template!
     template "rubocop.yml.tt", ".rubocop.yml"
     run_rubocop_autocorrections
 
+    # Remove the `test/` directory because we always use RSpec which creates
```

Comment:
It might be a good idea to do this before rubocop corrections since it'll give rubocop just a few less files it needs to read. Maybe just after project generation?

Reply:
Good idea. Will do.